### DISCUSSION TITLE
Punishment system ⚖️

### DISCUSSION THREAD

**Discussion by a7ssan** [Zrips/CMI#8007] [Feature Requests]
**We need a punishment system that can be synced with BungeeCord and has a customizable messaging feature.**
![image](https://github.com/Zrips/CMI/assets/44026213/2c02770a-b89f-4a50-bcfc-3b5ca8334b3e)

**Comment by mrfloris** [comment]
I think it's important that kicks and bans can be run on any server on the network, and applies to all the servers on the network. 
if an admin is on server 1, and player is on server 5, and the admin types /ban, or /kick, it should not be localized.
